### PR TITLE
Refactoring user administration

### PR DESCRIPTION
**Pull request by ngodfraind**
Fixed issues : 
https://github.com/claroline/Distribution/issues/3116

### CODE REVIEW

File: main/core/Resources/modules/administration/user-management/group/selectors.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const totalResults = createSelector(
+  [groups],
+  (groups) => groups.totalResults
+)
+
+export const select = {
+  groups,
+  data,
+  totalResults
+}
```
Comment (Elorfin) [inline main/core/Resources/modules/administration/user-management/group/selectors.js]: Those selectors already exists in `#/main/core/layout/list/selectors`

File: main/core/Resources/modules/administration/user-management/group/components/groups.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  PageContainer as Page,
+  PageHeader,
+  PageContent,
+  PageActions,
+  PageAction
+} from '#/main/core/layout/page/index'
+
+import {DataList} from '#/main/core/layout/list/components/data-list.jsx'
+
+class Groups extends Component {
```
Comment (Elorfin) [inline main/core/Resources/modules/administration/user-management/group/components/groups.jsx]: Do not use class when there is no internal state.

File: main/core/Resources/modules/enum/role.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export const PLATFORM_ROLE = 1
+export const WS_ROLE = 2
+export const CUSTOM_ROLE = 3
+export const USER_ROLE = 4
```
Comment (Elorfin) [inline main/core/Resources/modules/enum/role.js]: This is duplicated with `main/core/Resources/modules/administration/user-management/group/constants.js`.
Also, this file is in a wrong location : keep it in you app for now.

File: main/core/assets.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
     "entry": {
       "translator": "translation/browser",
       "organization-field": "symfony/form/organization/index",
-      "user-management": "users/module",
+      "user-management": "administration/user-management",
```
Comment (Elorfin) [inline main/core/assets.json]: Your module should be named `user` (not user-management).

File: main/core/Resources/modules/layout/alert/constants.js [hunk 5]
```diff
@@ -62,7 +62,7 @@ const ALERT_ACTIONS = {
   [actionConstants.ACTION_GENERIC]: {
     [ALERT_STATUS_PENDING]: {
       title: 'Chargement en cours',
-      message: 'Veuillez patienter pendant que nous chargeons vos données.'
+      message: 'Veuillez patienter pendant le chargement de vos données.'
```
Comment (LaurentGruber) [inline main/core/Resources/modules/layout/alert/constants.js]: ça ne devrait pas être dans un fichier de traduction ?

**Pull request merged**